graphql/resolvers.go

```diff
@@ -1064,6 +1067,49 @@ func (r *mutationResolver) SaveSubscription(ctx context.Context, subscription re
 	return true, nil
 }
 
+func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings *restModel.APIUserSettings) (bool, error) {
+	usr := route.MustHaveUser(ctx)
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return false, errors.Wrap(err, "Error retrieving Evergreen settings")
+	}
+	changedSettings, err := restModel.ApplyUserChanges(usr.Settings, *userSettings)
```

Comment:
If you're going to do this, it means that spruce will need to track just the changes made to the user settings object and only send those into the mutation. Are you sure that's what you want to do? I would think that sending the entire object would be easier

Reply:
Would there be an issue if i just passed back the entire user settings object into this function? Wouldnt it just overwrite every field anyways?